src/Estimators/EstimatorManagerNew.cpp

```diff
@@ -377,27 +377,27 @@ void EstimatorManagerNew::reduceOperatorEstimators()
     operator_recv_buffer.reserve(nops);
     for (int iop = 0; iop < operator_ests_.size(); ++iop)
     {
-      auto& estimator = *operator_ests_[iop];
-      auto& data = estimator.get_data_ref();
+      auto& estimator      = *operator_ests_[iop];
+      auto& data           = estimator.get_data_ref();
       size_t adjusted_size = data.size() + 1;
       operator_send_buffer.resize(adjusted_size);
       operator_recv_buffer.resize(adjusted_size);
-      auto cur   = operator_send_buffer.begin();
```

Comment:
These three lines are the fix.  Everything else is clang-format changing its mind about assignment alignment.